spec/features/items/my_items_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe "my items", type: :feature do
+  let(:password) { 'password' }
+  let(:user) { FactoryBot.create(:user, password: password) }
+  let(:group) { FactoryBot.create(:group) }
+
+  # for some reason, this does not work and fails with the following exception
+  # once i try to look at the membership in the debugger:
+  # "error: undefined method `status=' for Membership"
+
+  let(:membership) { FactoryBot.create(:membership, user: user, group: group, role: 1) }
+  let(:item1) { FactoryBot.create(:item, name: 'Item 1') }
+  let(:item2) { FactoryBot.create(:item, name: 'Item 2') }
```

Comment:
The feature itself should be implemented with the latest commit (16223c00be028a792e0521b1049931184a9ce253), but I can't seem to get the tests to be working. For some reason, creating a membership results in an exception, as soon as I try to take a look at it in the debugger. 

Would be really great, if one of you guys could take a look at it! @phkeese @yannik-dittmar